src/tracks/Pattern.cpp

```diff
@@ -752,8 +752,6 @@ PatternView::~PatternView()
 
 void PatternView::update()
 {
-	m_pat->changeLength( m_pat->length() );
```

Comment:
> It plays back two bars from the BB editor; it is the error described in #1105.

Yes, that's the old bug. The new one is that now you don't see it in the BBTrack until you reload the project unless you play it from the Song Editor.

> Should I apply #2964?

Not ready yet. But I can poke this one a bit and fix it in 2964? I need to rebase that one and try it with Sample Tracks any way.